internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -112,6 +135,7 @@ func convertSidecar(s map[string]*manifest.SidecarConfig) ([]*template.SidecarOp
 			EntryPoint:   entrypoint,
 			HealthCheck:  convertContainerHealthCheck(config.HealthCheck),
 			Command:      command,
+			PortMappings: portMappings[name],
```

Comment:
Why do we have both `Port` and `Portmappings` in `&template.SidecarOpts`

Reply:
Good question - sidecar `port` only defines the port that is being mentioned in the `port` field but portmappings hold all the extra ports as well which are being opened through either alb or nlb config. Does that make sense?

Reply:
Does this mean we don't need `Port` in `template.SidecarOpts` anymore?

Reply:
Yeah I think it is safe to remove Port from `template.SidecarOpts`.

Reply:
We will also remove Protocol from `template.SidecarOpts` as portmapping holds protocol info as well.